pkg/controllers/status/work_status_controller.go

```diff
@@ -92,6 +99,11 @@ func (c *WorkStatusController) Reconcile(ctx context.Context, req controllerrunt
 		return controllerruntime.Result{}, err
 	}
 
+	c.updateWorkDispatchingCondition(ctx, work)
+	if helper.IsWorkSuspendDispatching(work) {
+		return controllerruntime.Result{}, nil
+	}
```

Comment:
The `SuspendDispatching` indicates whether the controller should dispatch workload to the member cluster, it does not block status collection.  The reason why naming this field instead of a simple `Suspend` is to leave room for future extensions, like some users might not want to collect status probably due to performance concerns, then we can introduce another field like `suspendStatus`.

So, in my opinion, we should remove this check.

Reply:
I see. So we might as well add the condition in execution controller?

Reply:
Pushed up a commit, let me know what you think!

Reply:
> I see. So we might as well add the condition in execution controller?

Oh, yes. Thanks for the reminder, I didn't notice it yet.